homeassistant/components/withings/__init__.py

```diff
@@ -50,7 +50,7 @@
 )
 from .coordinator import WithingsDataUpdateCoordinator
 
-PLATFORMS = [Platform.BINARY_SENSOR, Platform.SENSOR]
+PLATFORMS = [Platform.EVENT, Platform.SENSOR]
```

Comment:
We should keep the old one for 6 months, deprecate it, and than remove it so users have time to transition

Reply:
I thought this was only with yaml stuff

Reply:
True. We should still users notice and some time to adapt as otherwise its pulling the rug out from under them and breaking their automations when we don't have a pressing need.

6 months seems to be the going timeframe now.

Reply:
Fixed